applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -93,13 +96,24 @@ start_builder(EndpointId, Member, Call) ->
 -spec resolve_endpoint_ids(wh_json:object(), whapps_call:call()) -> wh_proplist().
 resolve_endpoint_ids(Data, Call) ->
     Members = wh_json:get_value(<<"endpoints">>, Data, []),
-    [{Id, wh_json:set_value(<<"source">>, ?MODULE, Member)}
-     || {Type, Id, Member} <- resolve_endpoint_ids(Members, [], Data, Call)
-            ,Type =:= <<"device">>
-            ,Id =/= whapps_call:authorizing_id(Call)
-    ].
+    ResolvedEndpoints = resolve_endpoint_ids(Members, [], Data, Call),
+    FilteredEndpoints = [{Weight, {Id, wh_json:set_value(<<"source">>, ?MODULE, Member)}}
+                         || {Type, Id, Weight, Member} <- ResolvedEndpoints
+                            ,Type =:= <<"device">>
+                            ,Id =/= whapps_call:authorizing_id(Call)
+                        ],
+    Strategy = wh_json:get_value(<<"strategy">>, Data, ?DIAL_METHOD_SIMUL),
```

Comment:
This would be another reason to have a `strategy(Data)` function; so that accessing it is consistent within the module.